neurots/extract_input/input_distributions.py

```diff
@@ -85,20 +86,25 @@ def distributions(
 
     if diameter_input_morph is None:
         diameter_input_morph = filepath
-
-    if isinstance(diameter_model, str):
-        input_distributions["diameter"] = from_diameter.model(
-            load_morphologies(diameter_input_morph)
-        )
+    morphology = load_morphologies(diameter_input_morph)
+    if isinstance(diameter_model, str) and diameter_model.startswith("M"):
```

Comment:
Starts with "M"? Isn't it a bit arbitrary?

Reply:
it's name of internal neurots diametrizers, M1, M2, etc...